lib/main.dart

```diff
@@ -65,6 +65,7 @@ class LinShareApp extends StatelessWidget {
     return StoreProvider(
       store: store,
       child: MaterialApp(
+        debugShowCheckedModeBanner: false,
```

Comment:
Why do you disable this ?

Reply:
oh mb

Reply:
No worries, just asking because It doesn't bother IMO

Reply:
we have other ways to disable this in local only, so I should remove this.